src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -13,23 +36,45 @@ const CreateBenchmarkPage = () => {
 
   const addSubgoalMutation = trpc.iep.addSubgoal.useMutation();
 
+  const VIEW_STATES = {
+    BENCHMARK_PG_1: 0,
+    BENCHMARK_PG_2: 1,
+  };
+
+  const [viewState, setViewState] = useState(VIEW_STATES.BENCHMARK_PG_1);
+
+  const [benchmarkFormState, setBenchmarkFormState] =
+    useState<BenchmarkFormState>({
+      description: undefined,
+      setup: undefined,
+      materials: undefined,
+      instructions: undefined,
+      baseline_level: undefined,
+      target_level: undefined,
+      attempts_per_trial: undefined,
+      number_of_trials: undefined,
+    });
+
+  const steps = ["Instructional Guidelines", "Data Collection Guidelines"];
+
   const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
Hopefully we don't need it, but we should add a try/catch and some basic error handling here.